server/src/main/java/org/elasticsearch/search/rescore/QueryRescorerBuilder.java

```diff
@@ -45,7 +45,7 @@
     private static final ParseField RESCORE_QUERY_WEIGHT_FIELD = new ParseField("rescore_query_weight");
     private static final ParseField SCORE_MODE_FIELD = new ParseField("score_mode");
 
-    private static final ObjectParser<InnerBuilder, Void> QUERY_RESCORE_PARSER = new ObjectParser<>(NAME, null);
+    private static final ObjectParser<InnerBuilder, Void> QUERY_RESCORE_PARSER = new ObjectParser<>(NAME);
```

Comment:
All of these are because it couldn't compile now that there are two methods that take two args. It is better not to have the null anyway.